core/src/genesis.rs

```diff
@@ -83,22 +83,23 @@ pub fn genesis_testnet2() -> core::Block {
 			height: 0,
 			previous: core::hash::Hash([0xff; 32]),
 			timestamp: time::Tm {
-				tm_year: 2017 - 1900,
+				tm_year: 2018 - 1900,
 				tm_mon: 3,
-				tm_mday: 21,
-				tm_hour: 22,
+				tm_mday: 26,
+				tm_hour: 16,
```

Comment:
You seem to like early morning chaos, sure, why not? I guess this is when there's a good window of availability for all hands.